controllers/utils/copy_trigger_annotations.go

```diff
@@ -0,0 +1,134 @@
+/*
+Copyright 2024 The VolSync authors.
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU Affero General Public License as published
+by the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU Affero General Public License for more details.
+
+You should have received a copy of the GNU Affero General Public License
+along with this program.  If not, see <https://www.gnu.org/licenses/>.
+*/
+
+package utils
+
+import (
+	"time"
+
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+const (
+	// Annotation optionally set on src pvc by user.  When set, a volsync source replication
+	// that is using CopyMode: Snapshot or Clone will wait for the user to set a unique copy-trigger
+	// before proceeding to take the src snapshot/clone.
+	UseCopyTriggerAnnotation = "volsync.backube/use-copy-trigger"
+	CopyTriggerAnnotation    = "volsync.backube/copy-trigger"
+
+	// Annotations for status set by VolSync on a src pvc if UseCopyTriggerAnnotation is set to "true"
+	LatestCopyTriggerAnnotation             = "volsync.backube/latest-copy-trigger"
+	LatestCopyStatusAnnotation              = "volsync.backube/latest-copy-status"
+	LatestCopyTriggerWaitingSinceAnnotation = "volsync.backube/latest-copy-trigger-waiting-since"
+
+	// VolSync latest-copy-status annotation values
+	LatestCopyStatusValueWaitingForTrigger = "WaitingForTrigger"
+	LatestCopyStatusValueInProgress        = "InProgress"
+	LatestCopyStatusValueCompleted         = "Completed"
+
+	CopyTriggerWaitTimeout time.Duration = 10 * time.Minute
+)
```

Comment:
Let's move these out to `api/v1alpha1/common_types.go` since users and other controllers are expected to use these "well-known" names.